### DISCUSSION TITLE
Date ranges

### DISCUSSION THREAD

**Discussion by vomikan** [moneymanagerex/moneymanagerex#6111] [General]
What is a difference?
Since 1 Month Ago
1 Month Ago to Date

Please give examples in numbers.

**Comment by tactilis** [comment]
I assume you are referring to the discussion here: https://github.com/moneymanagerex/moneymanagerex/issues/6072#issuecomment-1633434675

If you go through that thread, you'll see that we are trying to get to a consistent naming of the time periods because there was inconsistency before.

For today (2023-07-15):

- `Since 1 Month Ago` will be from 2023-06-16 onwards (the end date is 9999-12-31)
- `1 Month Ago to Date` will be from 2023-06-16 to 2023-07-15
